lib/config/options/index.ts

```diff
@@ -200,6 +200,13 @@ const options: RenovateOptions[] = [
       type: 'string',
     },
   },
+  {
+    name: 'presetCache',
```

Comment:
@secustor @rarkins Should we use a more descriptive option? or move it to an experimental env?

Reply:
I would name it `cachePreset` that way it is grouped with the other cache related settings

Reply:
Maybe sharedPresetCache ?